python/essex-config/essex_config/sources/file_source.py

```diff
@@ -3,18 +3,25 @@
 import json
 from collections.abc import Callable
 from functools import cache
+from io import TextIOWrapper
 from pathlib import Path
-from typing import Any
+from typing import Any, cast
 
-import tomllib
+import toml
 import yaml
 
 from essex_config.sources.source import Source
 from essex_config.sources.utils import path_from_variable
 
+
+def _read_toml(stream: TextIOWrapper) -> dict[str, Any]:
+    content: bytes = cast(bytes, stream.read())
```

Comment:
TextIOWrapper.read() would return a str, not bytes.

Reply:
Hmm, this looks like an issue with the tests, I'll make these hit real files instead of mocking bytes